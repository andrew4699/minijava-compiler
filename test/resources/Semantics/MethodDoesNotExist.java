class T {

  public static void main(String[] args) {
    System.out.println(new S().cool());
  }
}

class S {
  int i;
  public int cool() {
    return this.doesntExist();
  }
}