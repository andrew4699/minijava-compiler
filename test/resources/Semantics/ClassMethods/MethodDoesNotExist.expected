[ClassMethodsVisitor - 11]: doesntExist() does not belong to class S
